cwl/samples_fillout_index_workflow.cwl

```diff
@@ -70,9 +88,20 @@ steps:
     scatterMethod: dotproduct
     out: [ bam_indexed ]
 
+  run_maf_filter:
```

Comment:
@nikhil @allanbolipata why are we doing the filter maf step here in the fillout workflow now?